python/desc/imsim/cosmic_rays.py

```diff
@@ -137,6 +138,47 @@ def read_catalog(catalog_file, ccd_rate=None, extname='COSMIC_RAYS'):
             cosmic_rays.ccd_rate = ccd_rate
         return cosmic_rays
 
+    def set_seed(self, seed):
+        """
+        Set the random number seed for a numpy.random.RandomState
+        instance that's held as the self.rng attribute.
+
+        Parameters
+        ----------
+        seed: int
+            The seed must be between 0 and 2**32 - 1
+        """
+        self.rng = np.random.RandomState(seed)
+
+    @staticmethod
+    def get_seed(visit, det_name):
```

Comment:
Perhaps this should be generate_seed ?

get_seed sounds more like a function to retrieve the current seed.

Reply:
Sure, I can change it to that.  I almost called it `make_seed`.